Project/ISVR/src/aakash.py

```diff
@@ -0,0 +1,124 @@
+"""
+Developer   : Aakash Panchal
+College ID  : 201801459
+About       : Added file and folder finder, which searches for a given file(or folder) among all the files or 
+              folders on the pc. In case there are multiple files or folders with
+              the same name, then it will print all the paths of found files or folders.
+Sources     : Python Documentation
+"""
+
+import os
+import platform
+import subprocess
+
+
+## -----------------------Folder finder START------------------------------------------
+def find_folder(name, path):
+    name = name.lower()
+    result = []
+    for root, dirs, files in os.walk(path,topdown = False):
+        dirs = [x.lower() for x in dirs]
+        if name in dirs:
+            result.append(os.path.join(root, name))
+    return result
+    
+# Protocol: "Open {folder_name} folder"

```

Comment:
For better use, you can simply ask the user to speak the folder otherwise we will have to figure out the generalized protocol. user 'from src.va import ask, speak' to import and then speak("please specify the folder name"), then ask().
otherwise, the functions are sound.